cmd/auth-handler.go

```diff
@@ -183,7 +187,7 @@ func getSessionToken(r *http.Request) (token string) {
 	if token != "" {
 		return token
 	}
-	return r.URL.Query().Get(xhttp.AmzSecurityToken)
+	return r.Form.Get(xhttp.AmzSecurityToken)
```

Comment:
This function is called in lines like:
```
	defer logger.AuditLog(ctx, w, r, mustGetClaimsFromToken(r))
```

It's not clear if `r.Form` is populated when the function executes or it uses the arguments' values when they are passed at call time - at this time ParseForm is not yet called.

Reply:
@donatello ParseForm is called in `setAuthHandler()` so there is nothing that is going to be missing.